crates/subspace-farmer/src/commitments.rs

```diff
@@ -45,14 +44,13 @@ struct Handlers {
         Bag<Arc<dyn Fn(&CommitmentStatusChange) + Send + Sync + 'static>, CommitmentStatusChange>,
 }
 
-#[derive(Debug)]
```

Comment:
Why was `Debug` removed? I believe all of the nested types have no issues with it.